release_notes/ocp_3_9_release_notes.adoc

```diff
@@ -677,6 +677,41 @@ As of {product-title} 3.9,
 xref:../upgrading/manual_upgrades.adoc#install-config-upgrading-manual-upgrades[manual
 upgrades] are not supported. In a future release, this process will be removed.
 
+[discrete]
+[[ocp-39-schedulable-masters]]
+=== Masters Marked as Schedulable Nodes by Default
+
+In previous versions of {product-title}, master hosts were marked as
+unschedulable nodes by default by the installer, meaning that new pods could not
+be placed on the hosts. Starting with {product-title} 3.9, however, masters are
+marked schedulable automatically during installation and upgrade. This change is
+mainly so that the web console, which used to run as part of the master itself,
+can instead be run as a pod deployed to the master.
+
+[discrete]
+[[ocp-39-default-node-selector]]
+=== Default Node Selector Set By Default and Automatic Node Labeling
+
+Starting in {product-title} 3.9, masters are now marked as schedulable nodes by
+default. As a result, the default node selector (defined in the master
+configuration file's `projectConfig.defaultNodeSelector` field to determine
+which node that projects will use by default when placing pods, and previously
+left blank by default) is now set by default during cluster installations and
+upgrades. It is set to `node-role.kubernetes.io/compute=true` unless overridden
+using the `osm_default_node_selector` Ansible variable.
+
+In addition, as long as `osm_default_node_selector` is left unset, any
+non-master, non-dedicated infrastructure nodes hosts (nodes with a
+`region=infra` label) defined in your inventory file will also have the
+`node-role.kubernetes.io/compute=true` label set automatically on them during
+installation. This ensures that the default node selector has available nodes to
+choose from when determining pod placement.
+
+If `osm_default_node_selector` is set to a custom value during installation or
```

Comment:
Per QE's testing in https://bugzilla.redhat.com/show_bug.cgi?id=1548641#c18, whatever osm_default_node_selector is set or NOT, this automatic node labeling always occurs. Master nodes are labeled with node-role.kubernetes.io/master=true, non-master, non-dedicated infrastructure nodes hosts (nodes with a region=infra label) are labeled with node-role.kubernetes.io/compute=true.